### PR TITLE
Fix Profile link from drawer on Mobile Web

### PR DESCRIPTION
**Pull request by ansh**
Fixes following bugs on mobile web:
- Active state was not registering with the bottom tab when on Profile
- Profile link in drawer was not working

### CODE REVIEW

**Review by estrattonbailey — APPROVED** [review]

**Comment by pfrazee** [comment]
I feel like this _shouldnt_ work just glancing at the code. The web uses a single flat navigation system, and this approach puts two routes on with the same path. My guess is that either `Profile` and `MyProfile` are going to get hit by the routes

**Comment by ansh** [comment]
@pfrazee It did work but it was such a hack. I created a new solution that is also a hack but a little better hack in my opinion since it isolates the hack into 3 lines of code

File: src/view/shell/Drawer.tsx [hunk 1]
```diff
@@ -64,6 +64,10 @@ export const DrawerContent = observer(function DrawerContentImpl() {
       const state = navigation.getState()
       store.shell.closeDrawer()
       if (isWeb) {
+        // hack because we have flat navigator for web and MyProfile does not exist on the web navigator -ansh
+        if (tab === 'MyProfile') {
+          navigation.navigate('Profile', {name: store.me.handle})
+        }
```
Comment (pfrazee) [inline src/view/shell/Drawer.tsx:70]: Need an else here, yeah?

**Review by pfrazee — CHANGES_REQUESTED** [review]
Okay yeah this looks better to me. One small question

File: src/view/shell/Drawer.tsx (hunk 1 above)
Comment (ansh) [reply src/view/shell/Drawer.tsx:70]: oops yeah thanks for the catch

Comment (ansh) [reply src/view/shell/Drawer.tsx:70]: Fixed! @pfrazee Pls approve so I can merge

Comment (ansh) [reply src/view/shell/Drawer.tsx:73]: @pfrazee Done, can you approve so I can merge?

**Review by pfrazee — APPROVED** [review]
🚀

**Pull request merged**